Libraries/LibGUI/GTextEditor.cpp

```diff
@@ -1297,10 +1316,22 @@ void GTextEditor::recompute_all_visual_lines()
 
 void GTextEditor::add_to_undo_stack(NonnullOwnPtr<UndoCommand> undo_command)
 {
-    if (m_undo_stack.size() <= m_undo_index)
-        m_undo_stack.append(make<NonnullOwnPtrVector<UndoCommand>>());
 
-    m_undo_stack[(m_undo_index)].insert(0, move(undo_command));
+    if (m_undo_stack_size <= 0) {
+        auto undo_commands_container = make<UndoCommandsContainer>();
+        undo_commands_container->m_undo_vector = {};
```

Comment:
You don't need to initialize this, it's already default-constructed on the line above it :)